pkg/minikube/cluster/mount.go

```diff
@@ -55,22 +57,25 @@ type mountRunner interface {
 }
 
 // Mount runs the mount command from the 9p client on the VM to the 9p server on the host
-func Mount(r mountRunner, source string, target string, c *MountConfig) error {
+func Mount(r mountRunner, source string, target string, c *MountConfig) {
 	if err := Unmount(r, target); err != nil {
-		return errors.Wrap(err, "umount")
+		exit.Error(reason.GuestMount, "mount failed", errors.Wrap(err, "umount"))
```

Comment:
lets not exit in here and return an error type like this
https://github.com/medyagh/minikube/blob/0e665930e86e118976b5c6b3912aaed51ccfe20f/pkg/drivers/kic/oci/errors.go?plain=1#L37

Reply:
Done.